corehq/apps/hqadmin/static/hqadmin/js/project_map.js

```diff
@@ -210,6 +210,18 @@ var projectMapInit = function(mapboxAccessToken) {
         info.update();
     }
 
+    function capitalizeCountryNames(countries) {
+        return countries.map(function(country) {
+            var formattedCountryName = country.charAt(0).toUpperCase();
+            if (country.indexOf(",") > -1) {
```

Comment:
@czue I agree. This is for formatting an array of country names that are in all caps, so it is a little misleading to say `capitalizeCountryNames`. I'll switch it back to `formatCountryNames`